src/rust/engine/src/lib.rs

```diff
@@ -75,6 +77,7 @@ use crate::handles::Handle;
 use crate::scheduler::{ExecutionRequest, RootResult, Scheduler, Session};
 use crate::tasks::{Rule, Tasks};
 use crate::types::Types;
+use crate::workunit_store::WorkUnitStore;
```

Comment:
```suggestion
use workunit_store::WorkUnitStore;
```

We generally use `crate::` to point at things which are defined in this crate, whereas `workunit_store` is an external crate.